cassandra/cassandra-docker.sh

```diff
@@ -286,7 +286,7 @@ if [ -n "$KEYSTORE_PASSWORD_FILE" ]; then
 fi
 if [ ! -n "$KEYSTORE_FILE" ]; then
     KEYSTORE_FILE="${KEYSTORE_DIR}/.keystore"
-    KEYSTORE_PASSWORD=${KEYSTORE_PASSWORD:-$(head /dev/urandom -c 512 | tr -dc A-Z-a-z-0-9 | head -c 17)}
+    KEYSTORE_PASSWORD=${KEYSTORE_PASSWORD:-$(openssl rand -base64 512 | tr -dc A-Z-a-z-0-9 | head -c 17)}
```

Comment:
How many bits/bytes of randomness do you need?  How long can the encoded value be?  For example:
```
>openssl rand 16 | od -c
0000000   8 275 362   h   a   ] 353 234 304 377 254   ~ 217 266   5   F
0000020
```
generates 16 binary bytes of randomness
```>openssl rand 16 -base64
//PkMtsmPZjqy/TTXSIklA==
>openssl rand 16 -base64|wc -c
25
```
This generates 16 bytes of randomness, but encodes it in 24 bytes (+1 for the newline).  The benefit is the shorter encoding.  The disadvantage is that you may have `=` padding if you do not use a multiple of 3 for the number of random bytes, and you will have non-alphanum like `+` and `/`.
```
>openssl rand 16 -hex
fbf3967a5d72ed8bb7b297de53c14230
>openssl rand 16 -hex|wc -c
33
```
This generates 16 bytes of randomness, but encodes it in 32 bytes (+1 for the newline).  The disadvantage is the longer encoding, but the advantage is only alphanum characters.